src/twisted/conch/ssh/transport.py

```diff
@@ -1906,7 +1906,8 @@ def _continue_KEX_ECDH_REPLY(ignored, hostKey, pubKey, signature):
         d.addCallback(_continue_KEX_ECDH_REPLY, hostKey, pubKey, signature)
         d.addErrback(
             lambda unused: self.sendDisconnect(
-                DISCONNECT_HOST_KEY_NOT_VERIFIABLE, b"bad host key"
+                DISCONNECT_HOST_KEY_NOT_VERIFIABLE,
+                f"bad host key [ecdh] {unused}".encode("utf-8"),
```

Comment:
Do we have automated tests for this code ?

Reply:
Apparently! I didn't have to put any coverage exception there :)